pkg/tcl/testworkflowstcl/testworkflowprocessor/operations.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2024 Testkube.
+//
+// Licensed as a Testkube Pro file under the Testkube Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//	https://github.com/kubeshop/testkube/blob/main/licenses/TCL.txt
+
+package testworkflowprocessor
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	testworkflowsv1 "github.com/kubeshop/testkube-operator/api/testworkflows/v1"
+)
+
+func ProcessDelay(_ InternalProcessor, layer Intermediate, container Container, step testworkflowsv1.Step) (Stage, error) {
+	if step.Delay == "" {
+		return nil, nil
+	}
+	t, err := time.ParseDuration(step.Delay)
+	if err != nil {
+		return nil, errors.Wrap(err, fmt.Sprintf("invalid duration: %s", step.Delay))
+	}
+	resources := map[corev1.ResourceName]intstr.IntOrString{
+		corev1.ResourceCPU:    {Type: intstr.String, StrVal: "50m"},
+		corev1.ResourceMemory: {Type: intstr.String, StrVal: "4Mi"},
```

Comment:
shouldn't it be configurable params?

Reply:
I don't think so, but maybe I'm wrong. It's for a very specific case of delay, that it's better to limit to the lowest possible resources.